### ISSUE TITLE
TEAM 1 - Make the corrections for the documentation from Milestone 1

### ISSUE THREAD

**Issue by print-Eruki** [uprm-inso4101-2023-2024-S1/semester-project-tutoring-app#119]
Last milestone we receive a relatively bad grade, this task involves making the suggested changes from the feedback received and receiving more feedback (TA's and professor).

Task points: 3 (Difficulty) * 9 (Priority) =  27 pts

**Comment by PinkSylvie** [comment]
@Ezequiel-Mendez I'm confused on what you mean with 65 points, 85 points and etc obtained. Does 0 points obtained mean that we didn't get any points on them since we didn't add those things? And what does the points mean regarding if to change or not to change?

**Comment by Ezequiel-Mendez** [comment]
The points that I put for each of the sections come from the feedback that was provided on the 1st milestone deliverable. I don't exactly know how these translate to percentages of the overall grade but I do think we should work them from the most that need work to the ones we did relatively well. 




![Image](https://github.com/uprm-inso4101-2023-2024-S1/semester-project-tutoring-app/assets/90644080/2b7c9948-86c3-4497-86a9-a1544965738f)

Here I have attached an image to the scores I am using. If anything let me know !
